databox/client/lib/DataboxExecutable.ts

```diff
@@ -53,7 +53,7 @@ export default class DataboxExecutable<ISchema extends IDataboxSchema = IDatabox
 
     try {
       databoxInternal.validateInput();
-      await this.plugins.onExec(databoxInternal, options, this.components.defaults);
+      await this.plugins.onStart(databoxInternal, options, this.components.defaults);
 
```

Comment:
I thought you didn't like using "start"?

Reply:
Dunno.. when did I say that? I just can't get exec vs run straight. It feels very confusing. I thought you suggested it could just be called onStart (and that was in the old docs, I think)

Reply:
I'm onboard with onStart.